src/unix/linux_like/linux/gnu/mod.rs

```diff
@@ -453,6 +453,45 @@ s! {
         pub aio_flags: ::__u32,
         pub aio_resfd: ::__u32,
     }
+
+    // netinet/tcp.h
+
+    pub struct tcp_info {
+        pub tcpi_state: u8,
+        pub tcpi_ca_state: u8,
+        pub tcpi_retransmits: u8,
+        pub tcpi_probes: u8,
+        pub tcpi_backoff: u8,
+        pub tcpi_options: u8,
+        // bitfields 4
+        pub tcpi_snd_wscale: u8,
+        // bitfields 4
+        pub tcpi_rcv_wscale: u8,
```

Comment:
The C header has these as 

```c
uint8_t	tcpi_snd_wscale : 4, tcpi_rcv_wscale : 4;
```

doesn't that mean `tcpi_snd_wscale` and `tcpi_rcv_wscale` are located within the same `u8`, and should be a single field in the Rust struct?

Reply:
I was trying to find what we do here in other cases, but it looks like this doesn't come up very often. Probably just make it one field and add a comment so users know:

```rust
/// The bitfield containing `tcpi_snd_wscale` and `tcpi_rcv_wscale`, four bits each.
pub tcpi_snd_rcv_wscale: u8
```

The musl section that is commented out should probably say something similar.